### ISSUE TITLE
Why is this dead???

### ISSUE THREAD

**Issue by relay3** [TeNNoX-Mods/Planetoids#12]
Was wondering if there was an up to date sphere mod (I was feeling creative as I like to create villages in really weird ways, like what TeNNoX describes in his post about villages here.), and I come across this wonderful mod!
Sadly, I also came across the latest version...
Fortunately, I found it was picked up by someone!
Sadly, the latest version for that was also really far back... ;-; halp.
Anyways, I'm wondering if TeNNoX will ever come back to these awesome ruins of a mod, or someone could make some up to date versions of it. (Preferably 1.11.2)

**Comment by relay3** [comment]
Or just a world gen

**Comment by tennox** [comment]
Yeah, well... I don't really have the time or motivation for this at the moment. I'm sorry. :disappointed:

**Comment by hyperdotbat** [comment]
I picked up the mod lately: https://www.curseforge.com/minecraft/mc-mods/planetoid

**Comment by tennox** [comment]
Nice! 😍
